caddy/Caddyfile.dev

```diff
@@ -0,0 +1,11 @@
+{
+	auto_https off
+}
+
+http://localhost:3000 {
+	handle_path /strapi* {
+		reverse_proxy http://strapi:8001
+	}
+
+	reverse_proxy http://app:3000
+}
```

Comment:
Out of curiosity, what is the difference between your config and the one provided as example on the Strapi docs? Don't we need the `strip_prefix` directive? Let me know, I haven't tested it.
```
http://example.com {
  route /test* {
    uri strip_prefix /test
    reverse_proxy 127.0.0.1:1337
  }
}
```

Reply:
`handle_path` implicitly strips the prefix (https://caddyserver.com/docs/caddyfile/directives/handle_path)